open_feature/open_feature_client.py

```diff
@@ -43,119 +43,103 @@ def get_boolean_value(
         key: str,
         default_value: bool,
         evaluation_context: EvaluationContext = None,
-        flag_evaluation_options: typing.Any = None,
```

Comment:
Flag evaluation options should remain on the client. The changes included in spec v0.4.0 was specifically focused on providers.